expo/expo.go

```diff
@@ -0,0 +1,91 @@
+package expo
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+// EjectMethod if the project is using Expo SDK and you choose the "plain" --eject-method those imports will stop working.
+type EjectMethod string
+
+// const ...
+const (
+	Plain   EjectMethod = "plain"
+	ExpoKit EjectMethod = "expoKit"
+)
+
+func (m EjectMethod) String() string {
+	return string(m)
+}
+
+// Expo CLI
+type Expo struct {
+	Version string
+	Method  EjectMethod
+}
+
+// New return a new *Expo with the provided version and --eject-method
+func New(version string, method EjectMethod) *Expo {
+	return &Expo{
+		Version: version,
+		Method:  method,
+	}
+}
+
+// InstallExpoCLI runs the install npm command to install the expo-cli
+func (e *Expo) InstallExpoCLI() error {
+	args := []string{"install", "-g"}
+	if e.Version != "latest" {
+		args = append(args, "expo-cli@"+e.Version)
+	} else {
+		args = append(args, "expo-cli")
+	}
+
+	cmd := command.New("npm", args...)
+	cmd.SetStdout(os.Stdout)
+	cmd.SetStderr(os.Stderr)
+
+	log.Printf("$ " + cmd.PrintableCommandArgs())
```

Comment:
this is a package, it shall not have hardcoded logging. please use logger to be able to disable/enable logging, or simply just return with the command and do the logging/running from main scope

Reply:
as far as I can see, almost all the functions is about to run commands. I think these functionalities are should not be a package but a well separated and managed main codebase.